pytorch_transformers/modeling_bert.py

```diff
@@ -214,6 +215,7 @@ def __init__(self,
             self.type_vocab_size = type_vocab_size
             self.initializer_range = initializer_range
             self.layer_norm_eps = layer_norm_eps
+            self.padding_idx = padding_idx
```

Comment:
If `padding_idx` is added to `BertConfig`, it should also be used in `BertEmbeddings`:
https://github.com/huggingface/pytorch-transformers/blob/e2a628a2233884921f08a225303e41fb4be58280/pytorch_transformers/modeling_bert.py#L249

Reply:
Thank you!  I agree with you. This is more general for arbitrary vocab. (which has another index as padding)